OWNERS_ALIASES

```diff
@@ -0,0 +1,10 @@
+# See the OWNERS docs at https://go.k8s.io/owners#owners_aliases
+
+aliases:
+  qe-approvers:
+  - juzhao
+  - lihongyan1
+  doc-approvers:
+  - bburt-rh
```

Comment:
after discussing with the docs team, we agreed that the engineering team could apply the docs-approved label themselves as long as the epic has already been ack-ed by the docs team.